Library/Homebrew/formulary.rb

```diff
@@ -166,8 +166,9 @@ def get_formula(spec)
   # * a formula pathname
   # * a formula URL
   # * a local bottle reference
-  def self.factory(ref, spec=:stable)
-    loader_for(ref).get_formula(spec)
+  def self.factory(ref, spec=:stable, is_installing=false, warn_all_ambiguity=false)
```

Comment:
This should instead take an `options` hash and use the last two parameter names as hash keys. This will improve readability of callers.